apps/portal/routers/pages.jag

```diff
@@ -69,6 +69,20 @@ var isMobileDevice = false;
         response.sendError(404, 'Requested page cannot be found');
         return;
     }
+
+    // Check whether a request for login and whether it is from secured channel
+    if(controller.localeCompare("/controllers/login-controller.jag") == 0 && !request.isSecure()){
+        var proc = require('process');
+
+        // If it is not secure redirect it to a secure channel
+        var newURL = request.getRequestURL().replace(request.getLocalPort(), proc.getProperty('mgt.transport.https.port'));
```

Comment:
Please do validate the above fix works when server starts with 443, and 80 port respectively.